tools/py/neb_interpolate.py

```diff
@@ -276,6 +287,12 @@ def spline_resample(q, nbeads_old, nbeads_new, k=3):
             print("Error: cannot find {}.".format(input_fin))
             exit(-1)
 
+        if np.all(inimasses == finmasses):
+            masses = inimasses
+        else:
+            print("Error: initial and final masses differ.")
```

Comment:
Could the error be made more transparent for the user? Perhaps with a suggestion on what they should do. 
Does this mean the initial and the final structures (two ends of the NEB) have different orders of atoms or something?